podcast/2.15.md

```diff
@@ -0,0 +1,1299 @@
+
+**Beyang Liu:**
+
+That's painful. Was there any work that you did back in those days that ended up being useful when you started working on Vitess and PlanetScale?
+
+**Sugu Sougoumarane:**
+
+Oh yes. So the big thing for me, which was really hard, I don't know if you know about the PayPal gang there, the founding PayPal members. They were all geniuses from UIUC.
+
+**Beyang Liu:**
+
+This is Peter Thiel, Elon Musk, and…
+
+**Sugu Sougoumarane:**
+
+It's the Max Levchin crowd.
+
+**Beyang Liu:**
+
+Okay. The Max Levchin crowd. Okay.
+
+**Sugu Sougoumarane:**
+
+Yeah, Max Levchin is from UIUC. And UIUC, a bunch of those people were from a school called IMSA, which is called International Math and Society Academy, which is kind of a school for gifted children or something.
+
+**Beyang Liu:**
+
+I see. So, super high IQ.
+
+**Sugu Sougoumarane:**
+
+Super high IQ. I was completely intimidated by the amount of brain power that was around me. And I had to figure out a way to prove my existence and justify that I'm useful, considering these people used to produce the kinds of features that they were producing. And so what I did was this: there was a problem that everyone hated. And it was actually the most problematic part of PayPal. There's an interesting story behind this.
+
+I don't know if you've read the old PayPal business model. Originally, when PayPal was founded, the way we raised money was we told them that we will allow people to send money to each other, and we will make money from the float. The fact that when people are sending money with each other means that we'll have some float money, and we'll make money off of that. So that was the business model that was proposed. You can probably find it if you search for it.
+
+But this was engineers writing code that moved money. They were inserting rows into a transaction table, updating the user balance, doing a commit, and moving on. And obviously–guess what happens–there are bugs in the software. So at some point in time, auditors came and said, "Okay, show us your books." And so they add up the transactions, they look at the user balances, and there's a few million dollars missing. And they're like, "Where's this money?" And we're like, "Oh, that's all in the float. It's float money. It's being invested somewhere."
+
+**Beyang Liu:**
+
+They didn't accept that answer.
+
+**Sugu Sougoumarane:**
+
+That kind of hand-waving doesn't go well in the financial industry. So, they basically asked us to account for every penny. And so somebody hacked up a tool. But because developers were writing software and there were bugs, that tool used to report discrepancies every day, or every other day. And every time it posted, somebody had to go in and investigate. And in millions of transactions, how do you find out where things went wrong? And so there was a victim engineer that was selected every once in a while the activity report reported conflicts. The guy said, "Okay, my life is gone today."
```

Comment:
The next to last sentence in this paragraph is a bit confusing - is there a way to clarify it with a bit of punctuation or gentle rewording?